friendsofopentelemetry/opentelemetry-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+    "copy-from-recipe": {
```

Comment:
`Error: You should register the bundle in the manifest as "friendsofopentelemetry/opentelemetry-bundle/1.0" is a Symfony bundle`